src/ui/map.js

```diff
@@ -1388,12 +1387,20 @@ class Map extends Camera {
         return [width, height];
     }
 
+    _hasCSS(): boolean {
+        for (const sheet of (window.document.styleSheets: any)) {
```

Comment:
How fast is this if a page has a bajillion CSS rules?

If iterating over them all in JS is expensive, another approach would be:

* Add `.mapboxgl-canary { background-color: salmon; }` to mapbox-gl.css. (Unfortunately `canary` is not a CSS color keyword so I chose another animal.)
* Map adds `<div class="mapboxgl-canary" style="visibility: hidden;"/>` and then checks the computed background color.